test/e2e/mock-e2e.js

```diff
@@ -886,6 +886,24 @@ async function setupMocking(
       };
     });
 
+  // Accounts API: transactions
+  await server
+    .forGet('https://accounts.api.cx.metamask.io/v1/accounts/0x5cfe73b6021e818b776b421b1c4db2474086a7e1/transactions')
```

Comment:
this one is added as a global mock because the account 0x5cf... is used in all specs, so we mock it for all